src/actionlist/action-list-item.scss

```
@@ -34,30 +34,44 @@
   }
 
   // only hover li without list as children
-  &:not(.ActionList-item--hasSubItem) {
+  &:not(.ActionList-item--hasSubItem),
+  // target contents of first child li if sub-item (li wraps item label + nested ul)
+  &.ActionList-item--hasSubItem > .ActionList-content {
     &:hover {
       cursor: pointer;
       background-color: var(--color-action-list-item-default-hover-bg);
+
+      // stylelint-disable-next-line selector-max-specificity
+      &:not(.ActionList-item--navActive) {
+        // stylelint-disable-next-line primer/box-shadow
+        box-shadow: inset 0 0 0 $border-width var(--color-action-list-item-default-active-border);
```

> @vdepizzol what do you think about using an `inset box-shadow` property for this?

Something to keep in mind is that `box-shadow` won't be visible in "Windows high contrast mode" and I think also not in the "Firefox custom color" mode.

I'm not sure if "Windows high contrast mode" is a requirement, but if we want to support it already and not switch to `border` (because it would require size changes) we could do the `box-shadow` + transparent `outline` combo, see [1. Custom focus styles + a transparent outline](https://sarahmhigley.com/writing/whcm-quick-tips/).

```suggestion
        box-shadow: inset 0 0 0 $border-width var(--color-action-list-item-default-active-border);
        // Support for "Windows high contrast mode"
        outline: $border-style $border-width 'transparent';
        outline-offset: -$border-width;
```

---

Oh, TIL! 🤗🤗

There's no reason we shouldn't support it! The transparent outline seems like a great solution while we don't have a unified `forced-colors` media query.